README.md

```diff
@@ -37,7 +37,7 @@ See the [changelog](CHANGELOG.md) for the latest release.
 You might also find useful information in the [Online XML Documentation](https://github.com/redhat-developer/vscode-xml/blob/master/docs/README.md) 
 or you can read this documentation inside vscode with the command `Open XML Document` available with `Ctrl+Shift+P`:
 
-![XML Open Documentation](docs/images/Commands/XMLCommands.png)
+![XML Open Documentation](https://raw.githubusercontent.com/redhat-developer/vscode-xml/master/docs/images/Commands/XMLCommands.png)
```

Comment:
It should work on embed doc (without Internet connection)

Reply:
Nope, as long as the extension is not installed you only have package.json + CHANGELOG + LICENSE.